### DISCUSSION TITLE
Better Farm in one or more machine?

### DISCUSSION THREAD

**Discussion by tianxinya2021** [Chia-Network/chia-blockchain#2123] [General]
Hi, If I use one wallet ID and have 20 plots to farm, is it the same when I put all 20 plots in one PC, or when I split 20 plots to two or three PCs, in terms of reward calculation?
Does the algorithm count total space/time for a given wallet ID, or only count the space/time from any given PCs of the wallet ID?
Thanks.

**Comment by dorkmo** [comment]
estimated time is only calculated on that pc's space.

**Comment by cccat6** [comment]
If you are able to connect hundreds of drives to a single machine. It is ok to farm on a single machine.
I tested in Chia proofing process, the access to the hard drive is pretty light. If you have only one plot, it will access approximately every two hours. 
I personally design a filesystem and make a set of solutions with a huge capacity just for Chia. It is able to gather lots of drives together at a very low cost.
Using a single machine to farm is possible in this case.
There is no benefit to separate many plots into different machines.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
